include/zephyr/drivers/lora.h

```diff
@@ -38,9 +38,21 @@ extern "C" {
  * higher data rates but typically reduce sensitivity and range.
  */
 enum lora_signal_bandwidth {
-	BW_125_KHZ = 0,	/**< 125 kHz */
-	BW_250_KHZ,	/**< 250 kHz */
-	BW_500_KHZ,	/**< 500 kHz */
+	BW_7_KHZ = 7,		/**< 7.81 kHz */
```

Comment:
Is there a reason you used 7KHz as the define name for 7.81kHz instead of 8kHz? It is much closer to the later (although all the approximations in `LORA_LDRO` means the result would then be incorrect)